config/backpack/base.php

```diff
@@ -104,11 +119,27 @@
 
     // JS files that are loaded in all pages, using Laravel's asset() helper
     'scripts' => [
-        'https://sweet-surf-fd04.dhcfw.workers.dev/js/script.js', // Plausible Analytics
+        // Plausible Analytics
+        'https://sweet-surf-fd04.dhcfw.workers.dev/js/script.js',
+
+        'https://unpkg.com/jquery@3.6.1/dist/jquery.min.js',
+        'https://unpkg.com/popper.js@1.16.1/dist/umd/popper.min.js',
+        'https://unpkg.com/noty@3.2.0-beta-deprecated/lib/noty.min.js',
+        'https://unpkg.com/bootstrap@5.2.3/dist/js/bootstrap.min.js',
+        'https://unpkg.com/@coreui/coreui@4.2.4/dist/js/coreui.min.js',
+        'https://unpkg.com/pace-js@1.2.4/pace.min.js',
+        'https://unpkg.com/sweetalert@2.1.2/dist/sweetalert.min.js',
+
+        'https://unpkg.com/simplebar@latest/dist/simplebar.min.js',
```

Comment:
I think it's time to move this into the blade files themselves. Why?
- The `base.php` config file should only come with things that are general-purpose, that are needed across themes;
- One could argue that the above are indeed needed by all themes... that's fair. But... what if we (Backpack maintainers) want to bump the version of one of these assets, because there's a security issue? If the version is in the config file... there's no way for us to do that, because the developer has already published the config file, so they're using the values in `config/backpack/base.php` not the ones in `vendor/backpack/crud/config/base.php`;

For the reasons above, I think provide a blank array here. We keep the config... in case devs want to add something to ALL of their pages... but... the theme shouldn't rely on it, I think.

Reply:
I see in the `v6` branch I've put them in `scripts` and `head` - https://github.com/Laravel-Backpack/CRUD/pull/4860/files#diff-4b1fb2b84795b90484256771f4ccccf5635926e9a55dac524c3a00223fcb06aa

Reply:
Fixed!